packages/server-wallet/src/__test-with-peers__/wallet/ensure-objectives.test.ts

```diff
@@ -94,7 +88,7 @@ describe('EnsureObjectives', () => {
 
     const result = await wallet.createChannels([getWithPeersCreateChannelsArgs(peerSetup)]);
 
-    expect(result).toBeObjectiveDoneType('EnsureObjectiveFailed');
+    await expect(result).toBeObjectiveDoneType('EnsureObjectiveFailed');
```

Comment:
Is there a lint rule that enforces that we `await` expected results about promises?

Reply:
That's a good question! I really hope so.

Reply:
There is a rule that enforces [await](https://eslint.org/docs/4.0.0/rules/require-await)  in async functions but unfortunately, it's not very nuanced. If a method is declared `async` you must have an `await` in it.

So something like `async () => someAsyncFunction()` would fail the linting.